src/app/content/components/ContentLink.tsx

```diff
@@ -54,6 +54,7 @@ export const ContentLink = (props: React.PropsWithChildren<Props>) => {
     ...anchorProps
   } = props;
   const {url, params} = getBookPageUrlAndParams(book, page);
+  const navigationMatch = createNavigationMatch(page as Page, book, params);
```

Comment:
you can't just cast this. either change the `createNavigationMatch` to only pick what it needs and hopefully that is only id and title or update Props here to add what `createNavigationMatch` needs